build.gradle

```diff
@@ -111,6 +106,29 @@ publishing {
             url = "$buildDir/localRepo"
         }
     }
+
+    publications {
+        mavenJava(MavenPublication) {
+            pom {
+                name = project.name
+                description = 'OpenSearch Software Development Kit'
+                url = "https://github.com/opensearch-project/opensearch-sdk-java"
+                licenses {
```

Comment:
Can you try this way https://github.com/opensearch-project/job-scheduler/blob/main/build.gradle#L107-L117 ?